content/kubermatic/main/architecture/concept/kkp-concepts/applications/default-applications-catalog/argocd/_index.en.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "ArgoCD Application"
```

Comment:
I think it makes sense to have the "Application" part in the page title because it helps contextualise the page, but for the navigation sidebar it's kind of bad.

Can you please also set `linkTitle` in the page metadata for all app pages and just use the application name (minus "Application")? This way, the navigation will look clean while the page title makes clear in what context "ArgoCD" is discussed here.

Reply:
Added `linkTitle` to all the pages. Thanks!